app/src/main/java/com/liato/bankdroid/appwidget/AutoRefreshService.java

```diff
@@ -94,6 +95,15 @@ private void handleStart(Intent intent, int startId) {
         }
     }
 
+    private boolean roamingDisable(NetworkInfo ni) {
```

Comment:
It would be great if this method name was updated to conform with java standards. e.g. `isUpdateOnRoaming()`